host/xtest/pkcs11_1000.c

```diff
@@ -8717,10 +8717,12 @@ static void xtest_pkcs11_test_1027(ADBG_Case_t *c)
 	if (!ADBG_EXPECT_CK_OK(c, rv))
 		goto out_unsetenv;
 
+	Do_ADBG_BeginSubCase(c, "Set ACL TEE client identity");
+
 	if (!ADBG_EXPECT_TRUE(c, !setenv("CKTEEC_LOGIN_TYPE", "group", 1)))
 		goto out_unsetenv;
 
-	str_len = snprintf(gid_str, sizeof(gid_str), "%x", getegid());
+	str_len = snprintf(gid_str, sizeof(gid_str), "%d", getegid());
```

Comment:
I agree. Libckteec gets group ID with [`strtoul(login_gid_env, &endp, 10);`](https://github.com/OP-TEE/optee_client/blob/master/libckteec/src/invoke_ta.c#L260-L268).
@vesajaaskelainen. Do you ack?